vendor/bat-native-confirmations/src/bat/confirmations/internal/time.cc

```diff
@@ -14,4 +14,19 @@ uint64_t Time::NowInSeconds() {
   return now.ToDoubleT();
 }
 
+uint64_t Time::MigrateToDoubleT(const uint64_t seconds) {
+  if (date_in_seconds < 10000000000) {
```

Comment:
Maybe I'm missing it, but what is `date_in_seconds`? Do you mean `seconds`?

Reply:
Refactored to be more concise